packages/data/tests/unit/adapters/facts/bard-test.ts

```diff
@@ -1077,6 +1077,107 @@ module('Unit | Adapter | facts/bard', function(hooks) {
     );
   });
 
+  test('urlForDownloadQuery', async function(assert) {
+    assert.expect(7);
+
+    assert.rejects(
+      //@ts-expect-error
```

Comment:
Why is this needed?

Reply:
It's testing that it throws an error if we pass in a v1 request. TS knows that this should be `requestVersion: '2.0'` and complains but we're expecting it

Reply:
Ah, yeah the interface supports it but not the implementation. I say we are V2 all the way and remove this test.

Reply:
Removed said test